mix.exs

```diff
@@ -0,0 +1,22 @@
+defmodule OpenTelemetry.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :opentelemetry_api,
+      version: "0.1.0",
+      elixir: "~> 1.9",
+      start_permanent: Mix.env() == :prod,
+      # We should never have dependencies
+      deps: [],
+      elixirc_paths: ["lib", "src"]
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
```

Comment:
Do we need `:logger` there?

Reply:
Hm, would use of Elixir `logger` greatly restrict versions of Elixir that will work since it will have to be one that can integrate with Erlang `logger`?

If so then we should remove that and in Elixir code simply call `:logger`.